docs/00_install/01_build-from-source/02_manual-build/index.md

```diff
@@ -0,0 +1,28 @@
+---
+content_title: EOSIO Manual Build
+---
+
+[[info | Manual Builds are for Advanced Developers]]
+| These manual instructions are intended for advanced developers. The [Shell Scripts](../01_shell-scripts/index.md) should be the preferred method to build EOSIO from source. If the script fails or your platform is not supported, continue with the instructions below.
+
+## EOSIO Dependencies
+
+When performing a manual build, it is necessary to install specific software packages that the EOSIO software depends on. To learn more about these dependencies, visit the [EOSIO Software Dependencies](00_eosio-dependencies.md) section.
+
+## Platforms
+
+Shell commands are available to manually download, build, install, test, and uninstall the EOSIO software and dependencies for these [platforms](03_platforms/index.md).
```

Comment:
We should avoid having a document for each variation of the platforms we support. We already discuss how to grab the dependencies with a dry-run of the script in `00_eosio-dependencies.md`. And this manual build section is really a plan C anyway (we currently recommend users to use binaries or use the script if they need source builds). 

So after the dependencies have been installed by the user, the remaining steps should be very simple and general to all the platforms we support. We should just include those directly in here. Take the "Build/Install/Test/Uinstall EOSIO" sections of the Ubuntu 18.04 unpinned instructions (which is the only platform we really need to claim we support for the manual build docs instructions at this point) and bring them in here (after making some simplifying changes which I note in the other document).

Reply:
In the interest of time, we can hold off on this change for this PR.